scripts/common.lib.ps1

```diff
@@ -117,6 +128,7 @@ function Locate-VsInstallPath($hasVsixExtension ="false"){
 
   $requiredPackageIds += "Microsoft.Component.MSBuild" 
   $requiredPackageIds += "Microsoft.Net.Component.4.6.TargetingPack"
+  $requiredPackageIds += "Microsoft.VisualStudio.Windows.Build"
```

Comment:
> $requiredPackageIds += "Microsoft.VisualStudio.Windows.Build" [](start = 1, length = 62)

can you help me understand why this package is required?

Reply:
scripts/build/TestFx.targets contains:
    <Import Project="$(MSBuildExtensionsPath)\Microsoft\WindowsXaml\v$(VisualStudioVersion)\Microsoft.Windows.UI.Xaml.CSharp.targets" Condition="$(TargetPlatformIdentifier) == 'UAP'" />

This targets file is part of the Microsoft.VisualStudio.Windows.Build component.  I ran into this because I have two instances of VS installed but the "latest" one has a very limited set of components.  The build script selected it because it was installed more recently and matched the other two component IDs, but the build failed because it couldn't find the targets file.